models/gif_cond_exp_multisynapse.cpp

```diff
@@ -393,6 +399,9 @@ nest::gif_cond_exp_multisynapse::~gif_cond_exp_multisynapse()
     gsl_odeiv_control_free( B_.c_ );
   if ( B_.e_ )
     gsl_odeiv_evolve_free( B_.e_ );
+
+  if ( S_.size_neuron_state_ > 0 )
```

Comment:
Bjarne Stroustrup actually recommends to test against zero-pointers as `p != 0` in C++ instead of comparing to `NULL` (which is recommended for C), since C++ type checking will ensure that whichever type `0` is it will be converted to pointer type before comparison so that the comparison is safe. `NULL` on the other hand, might be defined to be anything. See "The C++ Programming Language", Ch 5.1.1 *Zero*.